### ISSUE TITLE
2F authenication - mandatory

### ISSUE THREAD

**Issue by VINISHVV** [Ylianst/MeshCentral#2278]
Hi @Ylianst 

Is there any option to set 2F authentication mandatory for all users.

**Comment by Ylianst** [comment]
Yes, in the domain section of the config.json, add this:

```
      "passwordRequirements": {
        "force2factor": true
      }
```

For an example of a more advanced configuration, [look here](https://github.com/Ylianst/MeshCentral/blob/master/sample-config-advanced.json). The config.json [schema is here](https://github.com/Ylianst/MeshCentral/blob/master/meshcentral-config-schema.json).

**Comment by VINISHVV** [comment]
@Ylianst 

I have applied the same as per your suggestion. 

But still users can access Meshcentral without 2f auth. 

Please check below config and suggest.

__________________________________________
 "domains": {
    "": {
      "title": "Meshcentral",
      "_minify": true,
      "_newAccounts": true,
      "_userNameIsEmail": true,
      "_TitlePicture": "Muthoot-Finance-Logo.png",
      "_userallowedip" : "202.191.157.10,49.36.131.141"   
 },
  "passwordRequirements": {
        "force2factor": true,
         "reset": 90         
      },
      
___________________________________________________

**Comment by Ylianst** [comment]
They should be able to access the main page, but should not be able to create new device groups or access any devices until they setup 2FA. Can you confirm you see this behavior?

**Comment by VINISHVV** [comment]
Hi @Ylianst - new group creation is not allowed for the users. But they can access the all devices.

**Comment by Ylianst** [comment]
I did some fixes in MeshCentral v0.7.69 that was just published. Let me know if that fixes it.

**Comment by VINISHVV** [comment]
@Ylianst 

Its working fine now.
